java/util/pom.xml

```diff
@@ -88,6 +88,7 @@
                   <arg value="--proto_path=${protobuf.source.dir}"/>
                   <arg value="--proto_path=src/test/proto"/>
                   <arg value="src/test/proto/com/google/protobuf/util/json_test.proto"/>
+                  <arg value="src/test/proto/com/google/protobuf/util/json_test_proto2.proto"/>
```

Comment:
Addresses the following build failure:

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[49,53] package com.google.protobuf.util.proto.JsonTestProto2 does not exist
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1536,5] cannot find symbol
  symbol:   class TestAllTypesProto2
  location: class com.google.protobuf.util.JsonFormatTest
```

<details>
<summary>Full error snippet</summary>
<pre>
[INFO] --- compiler:3.6.1:testCompile (default-testCompile) @ protobuf-java-util ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 95 source files to /.../protobuf/java/util/target/test-classes
[INFO] /.../protobuf/java/util/target/generated-test-sources/protobuf_unittest/UnittestProto.java: Some input files use or override a deprecated API.
[INFO] /.../protobuf/java/util/target/generated-test-sources/protobuf_unittest/UnittestProto.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[49,53] package com.google.protobuf.util.proto.JsonTestProto2 does not exist
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1536,5] cannot find symbol
  symbol:   class TestAllTypesProto2
  location: class com.google.protobuf.util.JsonFormatTest
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1536,34] cannot find symbol
  symbol:   variable TestAllTypesProto2
  location: class com.google.protobuf.util.JsonFormatTest
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1539,38] cannot find symbol
  symbol:   variable TestAllTypesProto2
  location: class com.google.protobuf.util.JsonFormatTest
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1554,5] cannot find symbol
  symbol:   class TestAllTypesProto2
  location: class com.google.protobuf.util.JsonFormatTest
[INFO] 5 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Protocol Buffers [Parent] 4.27.0:
[INFO] 
[INFO] Protocol Buffers [BOM] ............................. SUCCESS [  0.000 s]
[INFO] Protocol Buffers [Parent] .......................... SUCCESS [  0.218 s]
[INFO] Protocol Buffers [Lite] ............................ SUCCESS [ 11.524 s]
[INFO] Protocol Buffers [Core] ............................ SUCCESS [ 17.252 s]
[INFO] Protocol Buffers [Util] ............................ FAILURE [  8.683 s]
[INFO] Protocol Buffers [Kotlin-Core] ..................... SKIPPED
[INFO] Protocol Buffers [Kotlin-Lite] ..................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  38.241 s
[INFO] Finished at: 2024-03-14T14:24:18-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.1:testCompile (default-testCompile) on project protobuf-java-util: Compilation failure: Compilation failure: 
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[49,53] package com.google.protobuf.util.proto.JsonTestProto2 does not exist
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1536,5] cannot find symbol
[ERROR]   symbol:   class TestAllTypesProto2
[ERROR]   location: class com.google.protobuf.util.JsonFormatTest
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1536,34] cannot find symbol
[ERROR]   symbol:   variable TestAllTypesProto2
[ERROR]   location: class com.google.protobuf.util.JsonFormatTest
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1539,38] cannot find symbol
[ERROR]   symbol:   variable TestAllTypesProto2
[ERROR]   location: class com.google.protobuf.util.JsonFormatTest
[ERROR] /.../protobuf/java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java:[1554,5] cannot find symbol
[ERROR]   symbol:   class TestAllTypesProto2
[ERROR]   location: class com.google.protobuf.util.JsonFormatTest
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :protobuf-java-util
</pre>
</details>